examples/cmd/vlan_name.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"errors"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/datatypes"
+	"github.com/softlayer/softlayer-go/session"
+)
+
+func init() {
+	vlanNameCmd.Flags().StringVarP(&VlanName, "name", "n", "", "New VLAN name")
+	vlanNameCmd.MarkFlagRequired("name")
+	rootCmd.AddCommand(vlanNameCmd)
+}
+
+var VlanName string
+var vlanNameCmd = &cobra.Command{
+	Use:   "vlan-name [vlanId]",
+	Short: "Set a VLAN's name",
+	Long:  `Set a VLAN's name.`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("a VLAN ID is required")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return VlanNameCommand(cmd, args)
+	},
+}
+
+func VlanNameCommand(cmd *cobra.Command, args []string) error {
+	// Sets up the softlayer-client session and Service
+	sess := session.New()
+	// sess.Debug = true
+	service := services.GetNetworkVlanService(sess)
+
+	// Make sure vlanId is an int
+	vlanId, err := strconv.Atoi(args[0])
+	if err != nil {
+		return err
+	}
+
+	// https://sldn.softlayer.com/reference/datatypes/SoftLayer_Network_Vlan/
+	// only Name and Note are editable on a Vlan object.

```

Comment:
Is it possible to edit the `note` datatype in this module too? It would be good to add this option too.